.github/workflows/release.yml

```diff
@@ -111,7 +111,13 @@ jobs:
             ARCHIVE_GOARCH=""
           fi
           ARCHIVE_NAME="consul-v${{ needs.set-product-version.outputs.product-version }}-${{ matrix.goos }}${ARCHIVE_GOARCH}.${ARCHIVE_EXTENSION}"
-          go build -ldflags="$GOLDFLAGS" -o ./out/$BINARY_NAME
+          
+          if [ "${{ matrix.goarch }}" == "arm64" ]; then
+            export CC=aarch64-linux-gnu-gcc
+          fi
+
```

Comment:
I don't see any fix for windows building here

Reply:
l119 fixes that.
CC prefix is required for arm only (and other exotic archs)